daliuge-engine/dlg/utils.py

```diff
@@ -179,6 +179,14 @@ def getDlgLogsDir():
     """
     return os.path.join(getDlgDir(), 'logs')
 
+def getDlgWorkDir():
+    """
+    Returns the location of the directory used by the DALiuGE framework to store
+    its logs. If `createIfMissing` is True, the directory will be created if it
+    currently doesn't exist
+    """
```

Comment:
What is this directory intended for? The docstring seems like a copy from the function above. From what I see this is meant to be the default working directory (but can still be overridden in the command line).